docs/messages/SYSTEM_ALARM.rst

```diff
@@ -0,0 +1,40 @@
+============
+SYSTEM_ALARM
+============
+
+This error tag corresponds to syslog messages notifying that there has been a change in status for an alarm.
+
+This error does not currently have a YANG model so uses ``NO_MODEL``.
+
+Implemented for:
+
+- junos
+
+Example:
+
+.. code-block:: json
+
+	{
+	  "message_details": {
+		"processId": "2449",
+		"hostPrefix": null,
+		"pri": "29",
+		"processName": "alarmd",
+		"host": "vmx01",
+		"tag": "Alarm set",
+		"time": "23:04:13",
+		"date": "Jul  8",
+		"message": "FPC color=RED, class=CHASSIS, reason=FPC 0 Major Errors"
+	  },
+	  "open_config": {
+		"hardware/classes/class/CHASSIS/componants/componant/FPC/state/color": "RED",
```

Comment:
This should be `components` instead of `componants` (please replace everywhere).